docs/Developers/Adding_CI.rst

```diff
@@ -0,0 +1,124 @@
+Adding CI to your Fork
+======================
+
+This tutorial will help you add the CI tools, that are used in
+coala repositories to test the code, to your forked repository.
+We recommend you to add all the CI and test everything in your
+own repository before doing a PR.
+
+Before we start adding CI its important you have a GitHub account
+and know how to fork repositories. In case you don't, you should
+have a look into our `Git Tutorial <https://coala.io/gitbasics>`_.
+
+Travis CI
+---------
+
+These are the official CI checks in travis :
+https://travis-ci.org/coala/coala/ and
+https://travis-ci.org/coala/coala-bears/.
```

Comment:
we have more repos with travis than those two, maybe mention that those are examples